### PR TITLE
Update README.md regarding the update font process

### PR DESCRIPTION
**Pull request by cardosogabriel**
Adding more detailed explanation on the update font process.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ This task will compile and minify the lesscss files into CSS files located at `d
 
 ### PatternFlyIcons Font
 
-PatternFlyIcons font is generated using [IcoMoon](http://icomoon.io/app).  [Load](http://icomoon.io/#docs/save-load) `PatternFlyIcons-webfont.json` as a new project in IcoMoon and update as necessary.  Please commit the updated `PatternFlyIcons-webfont.json` file in addition to the updated font files and supporting LESS/CSS changes.
+PatternFlyIcons font is generated using [IcoMoon](http://icomoon.io/app). Go to [manage projects](https://icomoon.io/app/#/projects) and import the project `PatternFlyIcons-webfont.json`. Load it and update as necessary. When finished, to again to manage projects, and download the updated `PatternFlyIcons-webfont.json` file. Also generate the fonts. Please commit both the updated `PatternFlyIcons-webfont.json` file, the updated font files and supporting LESS/CSS changes.
```
Comment (jeff-phillips-18) [inline README.md:123]: Please change:
When finished, to again to manage projects  -> When finished, return to manage projects

Please commit both the updated...  -> Please commit the updated...

**Comment by cardosogabriel** [comment]
Done! Does this work different than files, why don't I see changes in git status? I would squash, is there a way?

**Comment by dtaylor113** [comment]
Hi, squash is usually done via "git rebase -i <sha-id>"  where sha-id is the sha before the one you want to merge to.

**Comment by cardosogabriel** [comment]
Yes, that's what I usually do. But I don't event see the change when I do git status. How to proceed? Or can u guys merge it?

**Comment by jeff-phillips-18** [comment]
I definitely see both commits on your fork:

commit 94d5203cefb3f163496c1e51f2bd16db28d315f8
Author: Gabriel Cardoso gabrielc.cardoso@gmail.com
Date:   Thu Mar 17 15:29:52 2016 -0300

```
Update README.md
```

commit 5871e687be981e2e68e010b92399960acd00bbe7
Author: Gabriel Cardoso gabrielc.cardoso@gmail.com
Date:   Thu Mar 17 13:28:18 2016 -0300

```
Update README.md

Adding more detailed explanation on the update font process.
```

commit ee02b21b30ec62545e809dcba3bf70053ce3d068
Merge: c855124 402a282
Author: Dave Taylor dtaylor@redhat.com
Date:   Thu Mar 17 11:26:05 2016 -0400

```
Merge pull request #222 from cardosogabriel/PTNFLY-1309

PTNFLY-1309
```

...

**Pull request closed without merging**